components/infobox/wikis/warcraft/infobox_league_custom.lua

```diff
@@ -410,6 +410,10 @@ function CustomLeague._determineGame()
 	end
 
 	local startDate = _league:_cleanDate(_args.sdate) or _league:_cleanDate(_args.date)
+	if not startDate then
+		return
```

Comment:
No game is the expected value in this case, instead of eg. the default case of TFT?

Reply:
yeah defaulting to tft is better
adjusted